package.json

```diff
@@ -68,11 +69,17 @@
     "superagent": "^3.2.1",
     "through2": "^2.0.3"
   },
+  "optionalDependencies": {
+    "pelias-interpolation": "*"
```

Comment:
we probably should publish that module, since we publish everything else. it would make this nice and clean too. can we do that first and then update this PR?